src/features/create-token/CreateToken.types.ts

```diff
@@ -0,0 +1,15 @@
+import { MediaType } from '@zero-tech/zui/src/components/MediaInput';
+
+export interface DetailsFormSubmit {
+	mediaType: MediaType | undefined;
+	previewUrl: string;
+	avatar?: Buffer;
+	name: string;
+	symbol: string;
+}
+
+export interface TokenomicsFormSubmit {
+	tokenCount: string;
+	initialWalletAddress: string;
```

Comment:
nit: could probably have a clearer name for `initialWalletAddress` - I'm not 100% what that means from a glance, but that may just be a gap in my tokenomics knowledge.

Reply:
Yep I agree. This is vague. Updated to `initialTokenSupplyWalletAddress`.